### ISSUE TITLE
Change links to be relative

### ISSUE THREAD

**Issue by esilverm** [karansag/phrase-analogies-large-vae#1]
In [dataset_constructors/opposite_syntax_analogies.py](https://github.com/karansag/lingua52-project/blob/main/dataset_constructors/opposite_syntax_analogies.py), the links to files seem to be for files outside of the repository on your local device. Can we set this up with the files in the repo or as a way we can `wget` the files when we set up our tests, so people can easily replicate our findings?

In other words, where can we access the `snli_1.0_train.csv` file, and is there a way we can add it to the file structure of the repository? Also can we store the `Opposites_Dataset.csv` the same way?

If these files are too big for us to store on github, we can also add them to our `.gitignore`.
